scoring/score.py

```diff
@@ -53,8 +53,15 @@ def validate(self, extra_data):
                 f"Must be no more than {MAX_CANS} got {len(cans)}",
             )
 
-        # TODO: Check that teams are present if they are marked as leaving
-        # their zone
+        missing_but_moving_teams = []
+        for tla, info in self._teams_data.items():
+            if not info['present'] and info['left_scoring_zone']:
+                missing_but_moving_teams.append(tla)
```

Comment:
This miiight be neater as a comprehension?

Reply:
[af91950](https://github.com/srobo/sr2022-comp/pull/2/commits/af91950dbdd5bc647d502f4443db6eca9297c614)